### ISSUE TITLE
Formatting Code Wrong for Example

### ISSUE THREAD

**Issue by 3nder1ord** [diecode/KillerMoney#27]
In your example for Entities, where you show the color codes for Gold Ingots, you have an incorrect format. I found that using your example, the § symbol does not output correctly in the game. Using "\u00A72" instead of "§2" will correctly output the color code.

Also, if you can provide examples for different options for "meta-type: UNSPECIFIC", that would be great.

**Comment by kgahor** [comment]
Wrong output in the game? I haven't problem with this. Can you attach a screenshot? What is your minecraft version? Vanilla client?

**Comment by 3nder1ord** [comment]
Yes, sorry for not replying. I am using spigot 1.13.2 minecraft. I haven't
tested in 1.14 but I assume it will have the same problem.

Here is a screenshot of your example code:
[image: image.png]
Notice the "§2" is attached to the display name and the lore. When I used
this code, the game showed a box with a question mark.

Here's the code I use now:
[image: image.png]
Notice I used "\u00A73" instead of "§3". Using this, the game displayed the
color correctly. I don't know why it would't understand "§3", but in my
case at least, your example code didn't work.

**Comment by 3nder1ord** [comment]
I sent that as an email, so hopefully you can view the images in that.
